app/controllers/users/confirmations_controller.rb

```diff
@@ -4,22 +4,28 @@ class ConfirmationsController < Devise::ConfirmationsController
 
     def confirm
       with_unconfirmed_confirmable do
```

Comment:
this didn't change in this PR, but when I see `with_` and a block I usually think of something being yielded to in the block, ex:

```
with_unconfirmed_confirmable do |confirmable|
end
```

rather than our current `@confirmable` instance variable -- I think it's easier to follow. WDYT? Would there be any objections if I made a second PR to make that more explicit?